include/swift/Remote/MetadataReader.h

```diff
@@ -2932,34 +2915,64 @@ class MetadataReader {
     if (!typeDecl)
       return BuiltType();
 
+    // Attempt to build the parent (if it exists).
+    BuiltType parent;
+    if (auto parentDescriptor = readParentContextDescriptor(descriptor))
+      if (parentDescriptor->isResolved())
+        if (auto resolvedParent = parentDescriptor->getResolved())
+          parent = readNominalTypeFromDescriptor(metadata, resolvedParent);
```

Comment:
Not sure if this is ok, I'm passing the metadata of the current descriptor, not the parent.